src/index.ts

```diff
@@ -4,7 +4,17 @@ import fastifyPlugin from "fastify-plugin";
 import { createOnRequestHook } from "./hooks/onRequest";
 import { createOnSendHook } from "./hooks/onSend";
 
-export interface PluginOptions {
+/**
+ * Plugin options for fastify-aws-dynamodb-cache.
+ *
+ * @property {string} dynamoDbRegion - AWS region for DynamoDB client configuration.
+ * @property {string} [dynamoDbAddress] - Optional custom endpoint (useful for local development/testing).
+ * @property {string} tableName - DynamoDB table name used for storing cache entries.
+ * @property {number} defaultTTLSeconds - Global default TTL (in seconds) for all cache entries.
+ * @property {boolean} [disableCache=false] - If true, disables caching plugin-wide.
+ * @property {string} [passthroughQueryParam] - If defined, this query parameter will bypass cache when present in a request.
```

Comment:
What's the difference between wrapping the variable in "[]" and not doing it?

Reply:
It's a way to say that value is optional